libraries/RC_Channel/RC_Channel.h

```diff
@@ -99,6 +99,9 @@ class RC_Channel {
     void init_aux();
     bool read_aux();
 
+    static const uint16_t RC_LOW_DEFAULT = 1300;
```

Comment:
Not variables.  It's right there in the declaration :-P

Also not sure what the "DEFAULT" is in aid of.

I'd also point out that where these are used we should really be using `AUX_PWM_TRIGGER_LOW` and `AUX_PWM_TRIGGER_HIGH`.  Sadly, these are different values to the constants here.  When we move those functions to be RC_OPTIONs we run the risk of breaking people's setups until they fix their transmitter ranges :-(